templates/case-studies/templates/_macros-image.jinja

```diff
@@ -0,0 +1,18 @@
+# Params
+# image_url: URL of image (required)
+# caption: Caption for image
+
+{%- macro image(image_url, caption) -%}
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <div class="row">
+        <div class="u-align--center u-vertically-center u-no-padding--bottom p-image-container--cinematic">
+          <img src={{ image_url }} alt={{ caption }} />
```

Comment:
I think we need to include another param for the alt text, as in the example at `/case-studies` the alt is the copyright information.